backend/document-service/src/main/kotlin/com/nosql/document/dto/DocumentDto.kt

```diff
@@ -4,6 +4,7 @@ import com.fasterxml.jackson.annotation.JsonFormat
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.nosql.document.constants.jackson.DATE_PATTERN
 import com.nosql.document.enumerator.DocumentStatus
+import com.nosql.document.enumerator.DocumentStatus.ORDERED
 import com.nosql.document.enumerator.DocumentType
 import java.time.Instant
 import java.util.Date
```

Comment:
make this `DocumentDto` as `data class`

Reply:
It make no sense, we do not user toString